### DISCUSSION TITLE
multiple matchs

### DISCUSSION THREAD

**Discussion by physnoct** [hlorenzi/customasm#148] [Q&A] [answered]
How can we use mnemonics that are similar to one with space?

Here's an example:
#ruledef
{
	rld		=> 0xed @ 0x6f
	rl	d	=> 0xcb @ 0x12
}

test:
	rld
	rl	d

The assembler fails to resolve the instruction even when there is a space.

**Comment by physnoct** [comment, marked as answer]
There's already an issue about this.

**Comment by hlorenzi** [comment]
Yeah, it's a long-standing issue... It's a big problem 😅
